tests/test_flobject.py

```diff
@@ -662,10 +662,7 @@ def test_accessor_methods_on_settings_object(load_static_mixer_case):
 
     existing = solver.file.read.file_type.get_attr("read-only?", bool)
     modified = solver.file.read.file_type.is_read_only()
-    if solver.get_fluent_version() < "23.2.0":
-        assert existing == modified
-    else:
-        assert existing == None and modified == False
+    assert existing == modified
```

Comment:
@seanpearsonuk Here we have resolved 1 bug. Earlier it was returning `None` and `False` for same Boolean class type. Now both returns False.

![image](https://github.com/ansys/pyfluent/assets/106588300/d9321802-bc39-4995-af6d-6d1a91343232)